internal/writer/queryresults.go

```
@@ -0,0 +1,77 @@
+	"github.com/rayakame/sqlc-gen-better-python/internal/utils"
+)
+
+type QueryResultsWriter struct {
+	writer *CodeWriter
+}
+
+func NewQueryResultsWriter(writer *CodeWriter) *QueryResultsWriter {
+	return utils.ToPtr(QueryResultsWriter{writer: writer})
+}
+
+func (w *QueryResultsWriter) WriteQueryResultsClassHeader(connType string, initFields []string, driverReturnType string, async bool) {
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(golines)